engine-api/src/main/java/pl/edu/icm/unity/engine/api/translation/SystemTranslationProfileProvider.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2014 ICM Uniwersytet Warszawski All rights reserved.
+ * See LICENCE.txt file for licensing information.
+ */
+package pl.edu.icm.unity.engine.api.translation;
+
+import java.util.Map;
+
+import pl.edu.icm.unity.types.translation.ProfileType;
+import pl.edu.icm.unity.types.translation.TranslationProfile;
+
+/**
+ * Implementations (must be Spring components) provide additional system
+ * translation profiles, defined for standard providers. Those profiles are not
+ * saved in DB, but @{TranslationProfileManagement} also returns them. All
+ * system profiles should be defined as read only profile
+ * 
+ * @author P.Piernik
+ *
+ */
+public interface SystemTranslationProfileProvider
```

Comment:
Does it makes sense to have such interface? Shouldn't we have just InputTranslationProfileProvider and the Output one? I don't think that we even need interfaces - we can just have 2 such simple classes + 1 shared code. Preferably not an abstract parent but a collaborator (utility) class - whenever this is easy we prefer composition over inheritance.